packages/augur-tools/scripts/flash/index.js

```diff
@@ -115,13 +116,9 @@ var methods = {
         required: true,
         short: "o",
         help:
-          'Outcome, sets outcome to use, can be overridden by invalid flag, negative outcome use \\"-10\\"'
-      },
-      invalid: {
-        flag: true,
-        short: "i",
-        help: "Overrides outcome to pass invalid"
+          'Outcome, sets outcome to use, negative outcome use \\"-10\\"'
       },
+      description: { default: "", short: "d", help: "Add description" },
```

Comment:
What is an "invalid" description?

Reply:
Should it be aliased to something like outcome id?

Reply:
new string param on contracts. I default the value in next PR.